CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the [engine CONTRIBUTING file](https://github.com/OpenTermsArchive/engine/blob/main/CONTRIBUTING.md#changelog). This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## Unreleased [patch]
+
+_Full changeset and discussions: [#3](https://github.com/OpenTermsArchive/engine/pull/3)._
+
+> Development of this release was supported by the [French Ministry for Foreign Affairs](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/diplomatie-numerique/) through its ministerial [State Startups incubator](https://beta.gouv.fr/startups/open-terms-archive.html) under the aegis of the Ambassador for Digital Affairs.
+
+### Added
+
+- Add warning log for disabled Logger mailer due to missing config
```

Comment:
> This is accurate; I just suggest focusing on the impact for end users rather than on the technical description: “log emails cannot be sent” rather than “disabled Logger mailer”.
```suggestion
- Log a warning in case log emails cannot be sent because of a missing config
```